packages/webpack-plugin/src/index.ts

```diff
@@ -42,7 +53,10 @@ function webpackDebugIdInjectionPlugin(): UnpluginOptions {
         new BannerPlugin({
           raw: true,
           include: /\.(js|ts|jsx|tsx|mjs|cjs)$/,
-          banner: () => getDebugIdSnippet(uuidv4()),
+          banner: (arg?: BannerPluginCallbackArg) => {
+            const debugId = arg?.chunk?.hash ? stringToUUID(arg?.chunk?.hash) : uuidv4();
```

Comment:
when is the hash not available? 

If so is this even truely deterministic? Is there another string we can fallback on if the hash does not exist?

can we get a test to confirm?

Reply:
From the type def:

![Screenshot 2023-06-15 at 16 53 54](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/8118419/585495ee-0c75-4278-ace6-6c69d66627de)

I honestly more or less did this out of caution. Generally, it should be available though.

> If so is this even truely deterministic?

Should be, yes.

>  Is there another string we can fallback on if the hash does not exist?

I didn't investigate yet. For now I thought this was a quick win that works 99% of the time.

Reply:
Test I will add!